locale/da/texts.po

```diff
@@ -31,44 +31,44 @@ msgstr "Sendt ${ symbol }"
 msgid "You sent ${ symbol } to yourself"
 msgstr "Du sendte ${ symbol } til dig selv"
 
-#. See https://momentjs.com/docs/#/displaying/calendar-time/
+#.  See https://momentjs.com/docs/#/displaying/calendar-time/
 #: src/models.js:41
 msgid "[Today •] HH:mm"
 msgstr "[I dag •] HH:mm"
 
-#. See https://momentjs.com/docs/#/displaying/calendar-time/
+#.  See https://momentjs.com/docs/#/displaying/calendar-time/
 #: src/models.js:42
 msgid "[Tomorrow •] HH:mm"
 msgstr "[I morgen •] HH:mm"
 
-#. See https://momentjs.com/docs/#/displaying/calendar-time/
+#.  See https://momentjs.com/docs/#/displaying/calendar-time/
 #: src/models.js:43
 msgid "dddd [•] HH:mm"
 msgstr "dddd [•] HH:mm"
 
-#. See https://momentjs.com/docs/#/displaying/calendar-time/
+#.  See https://momentjs.com/docs/#/displaying/calendar-time/
 #: src/models.js:44
 msgid "[Yesterday •] HH:mm"
 msgstr "[I går •] HH:mm"
 
-#. See https://momentjs.com/docs/#/displaying/calendar-time/
+#.  See https://momentjs.com/docs/#/displaying/calendar-time/
 #: src/models.js:45
 msgid "[Last] dddd [•] HH:mm"
 msgstr "[Sidste] dddd [•] HH:mm"
 
-#. See https://momentjs.com/docs/#/displaying/calendar-time/
+#.  See https://momentjs.com/docs/#/displaying/calendar-time/
 #: src/models.js:46
 msgid "DD MMM YYYY [•] HH:mm"
 msgstr "DD MMM YYYY [•] HH:mm"
 
-#: src/utils.js:144
+#: src/utils.js:145
 msgid "Invalid address"
 msgstr "Ugyldig adresse"
 
 #. *
-#. * @example
-#. * You have received 10 T2, 5 T1 and 2 other token on a new transaction.
-#.
+#.          * @example
+#.          * You have received 10 T2, 5 T1 and 2 other token on a new transaction.
+#.          
```

Comment:
Curious whitespace, this file was generated automatically, right?

Reply:
Right!